fw/aux_port.h

```diff
@@ -148,6 +153,14 @@ class AuxPort {
           status_.spi.nonce += 1;
           break;
         }
+        case SampleType::kCuiAmt22: {
+          int updated = cui_amt22_->ISR_Update(&status_.spi.value);
```

Comment:
Assuming this is a bool, `ISR_Update` should return a bool and this variable should be `const`.  Using `auto` is also fine, but not required.